chips/cc26xx/src/aon.rs

```diff
@@ -0,0 +1,41 @@
+//! Always On module
+
+use kernel::common::VolatileCell;
+
+#[repr(C)]
+pub struct AonEventRegisters {
+    mcu_wu_sel: VolatileCell<u32>,       // MCU Wake-up selector
+    aux_wu_sel: VolatileCell<u32>,       // AUX Wake-up selector
+    event_to_mcu_sel: VolatileCell<u32>, // Event selector for MCU Events
+    rtc_sel: VolatileCell<u32>,          // RTC Capture event selector for AON_RTC
+}
+
+pub struct AonEvent {
+    regs: *mut AonEventRegisters,
+}
+
+const AON_EVENT_BASE: *mut AonEventRegisters = 0x4009_3000 as *mut AonEventRegisters;
```

Comment:
Move the constant into the struct and there is no need for the pointer to be `*mut` i.e, change it to `*const`

Reply:
Sure, will do! :) Had no idea it'd work as a const, good to know!